tnr/plugins/sesameproxy.py

```diff
@@ -39,18 +43,44 @@ def resolve(self,name):
 
         try:
             source_coord = SkyCoord(result_table['RA'],result_table['DEC'],unit=("hourangle","deg"))
+            object_type = str(result_table[0]['OTYPE']).strip()
+            # query rdf ivoa data
+            ivoa_ttl_path = os.environ.get("IVOA_RDF_DATA", None)
+            object_links = None
+            if ivoa_ttl_path is not None:
+                links = None
+                G = rdflib.Graph()
+                G.parse(ivoa_ttl_path, format="ttl")
+                label_matches_list = list(G[:rdflib.URIRef('http://www.w3.org/2000/01/rdf-schema#label'):rdflib.Literal(object_type)])
+                if len(label_matches_list) == 1:
+                    exact_matches_links = list(G[label_matches_list[0]:rdflib.URIRef('http://www.w3.org/2000/01/rdf-schema#exactMatch')])
+                    for link in exact_matches_links:
+                        if links is None:
+                            links = []
+                        links.append(str(link))
```

Comment:
would something like this work?
```suggestion
            if ivoa_ttl_path is not None:
                links = []
                G = rdflib.Graph()
                G.parse(ivoa_ttl_path, format="ttl")
                for label_match in G[:rdflib.URIRef('http://www.w3.org/2000/01/rdf-schema#label'):rdflib.Literal(object_type)]:                
                    links.extend([str(link) for link in G[label_matches_list[0]:rdflib.URIRef('http://www.w3.org/2000/01/rdf-schema#exactMatch')]])
```

also I am not too sure why you need `object_links` variable, it may be possible to pass links directly?